filament/backend/include/backend/DriverEnums.h

```diff
@@ -1219,6 +1222,20 @@ struct StencilState {
     uint8_t padding = 0;
 };
 
+struct PushConstant {
+    using Variant = std::variant<int, float, bool>;
+    char const* name = nullptr;
```

Comment:
In this header the push constant should only be a value, it shouldn't need to have a name.  The name is needed in Program though, so that the gl backend can find it in the shader.

Reply:
removed.